File: app/src/main/java/com/pietrantuono/podcasts/addpodcast/singlepodcast/model/PodcastSchedulers.java
package com.pietrantuono.podcasts.addpodcast.singlepodcast.model;

import rx.Scheduler;
import rx.android.schedulers.AndroidSchedulers;
import rx.schedulers.Schedulers;

public class PodcastSchedulers {
    
    public Scheduler mainThread() {
        return AndroidSchedulers.mainThread();
    }

    public Scheduler newThread() {
        return Schedulers.newThread();
    }
}

File: app/src/main/java/com/pietrantuono/podcasts/addpodcast/singlepodcast/model/PodcastSchedulers.kt
package com.pietrantuono.podcasts.addpodcast.singlepodcast.model

import io.reactivex.Scheduler

import io.reactivex.android.schedulers.AndroidSchedulers
import io.reactivex.schedulers.Schedulers

open class PodcastSchedulers {

    open fun mainThread(): io.reactivex.Scheduler? {
        return AndroidSchedulers.mainThread()
    }

    open fun newThread(): Scheduler {
        return Schedulers.newThread()
    }
}
